modules/navigation_3d/register_types.cpp

```diff
@@ -65,6 +65,7 @@ void initialize_navigation_3d_module(ModuleInitializationLevel p_level) {
 #ifdef TOOLS_ENABLED
 	if (p_level == MODULE_INITIALIZATION_LEVEL_EDITOR) {
 		EditorPlugins::add_by_type<NavigationRegion3DEditorPlugin>();
+		EditorPlugins::add_by_type<NavigationObstacle3DEditorPlugin>();
```

Comment:
You need to include it here.

Reply:
yep, the joy of rebasing with lines just disappearing by accident.